CMakeLists.txt

```diff
@@ -195,13 +177,8 @@ if(${FILESYSTEM_FOUND})
     set(CMAKE_CXX_STANDARD 17)
```

Comment:
Per my Request: Let's just move these up to the STANDARD definitions ; that simplifies the current `if/else` to just `if(NOT FILESYSTEM_FOUND` ... no need to `${}` around it either, BTW

Reply:
Agreed. Fixed it.